packages/environment-ember-template-imports/-private/environment/preprocess.ts

```diff
@@ -1,54 +1,91 @@
 import { GlintExtensionPreprocess } from '@glint/core/config-types';
-import { parseTemplates } from 'ember-template-imports';
 import { GLOBAL_TAG, PreprocessData, TemplateLocation } from './common';
 
 const TEMPLATE_START = `[${GLOBAL_TAG}\``;
 const TEMPLATE_END = '`]';
 
+// content-tag 1.2.2:
+//   The current file is a CommonJS module whose imports will produce 'require' calls;
+//   however, the referenced file is an ECMAScript module and cannot be imported with 'require'.
+//   Consider writing a dynamic 'import("content-tag")' call instead.
+//   To convert this file to an ECMAScript module, change its file extension to '.mts',
+//   or add the field `"type": "module"` to 'glint/packages/environment-ember-template-imports/package.json'.ts(1479)
+//
+// ...Except,
+//    > the referenced file is an ECMAScript module
+//
+//    package.json#exports does refer to a cjs file if required, so TS should be resolving the `require`
+//    entries not the `import` entries.
+//
+//    https://github.com/embroider-build/content-tag/blob/v1.2.2-content-tag/package.json#L13-L21
+//
+// @ts-expect-error see above
```

Comment:
even though this ts-expect-error is here, we still get the correct type for `Preprocessor` and `p.parse` below

Reply:
Interesting. I wonder why I don't get this in the prettier plugin.

Reply:
is it using 1.2.2 / same tsconfig settings from glint?